src/schema/v2/partnerDocumentsConnection.ts

```diff
@@ -0,0 +1,89 @@
+import config from "config"
+import {
+  GraphQLFieldConfig,
+  GraphQLInt,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
+import {
+  connectionWithCursorInfo,
+  paginationResolver,
+} from "schema/v2/fields/pagination"
+import { ResolverContext } from "types/graphql"
+import { InternalIDFields } from "./object_identification"
+
+const PartnerDocumentType = new GraphQLObjectType<any, ResolverContext>({
+  name: "PartnerDocument",
+  fields: {
+    ...InternalIDFields,
+    uri: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    filename: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    title: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    size: {
+      type: new GraphQLNonNull(GraphQLInt),
+    },
+    publicUrl: {
```

Comment:
**suggestion**: `publicURL`

Reply:
**question**: Are all of these fields used in the design? Wondering if we could get by with just `internalID`, `title`, and `publicURL` to start...